arm/entrypoint.sh

```diff
@@ -2,60 +2,77 @@
 
 set -e
 
-# Create Resource group if it does not exists
-
-RESOURCE_GROUP_EXISTS=$(az group exists -n "$AZURE_RESOURCE_GROUP")
-if [[ $RESOURCE_GROUP_EXISTS == "false" ]]
+if [[ -z "$AZURE_RESOURCE_GROUP" ]]
 then
-  if [[ -z "$RESOURCE_GROUP_LOCATION" ]]
+  echo "AZURE_RESOURCE_GROUP is not set." >&2
+  exit 1
+else
```

Comment:
Do you need this else. Meaning since you are exiting from if above, you can avoid the nesting

Reply:
yes this can be avoided, will make the change